extensions/wikia/ARecoveryEngine/js/SourcePoint/mms_client.js

```diff
@@ -0,0 +1,71 @@
+(function() {
+
+    window._sp_ = window._sp_ || {};
+   
+    // Init MMS client cmd queue
+    window._sp_.mms = window._sp_.mms || {};
+    window._sp_.mms.cmd = window._sp_.mms.cmd || [];
+    window._sp_.mms._internal = window._sp_.mms._internal || {};
+
+    // Init MMS API
+    var targetingParams = {};
+    window._sp_.mms.setTargeting = function(key, value) {
+        targetingParams[key] = value;
+    };
+
+    window._sp_.mms.startMsg = function() {
+        window._sp_.messagingDetection(loadMessage);
+    };
+
+    window._sp_.mms._internal.cdc1 = function() {
+        if (typeof window._sp_.config.mms_client_data_callback === 'function') {
+            try {
+                window._sp_.config.mms_client_data_callback.apply(
+                    window._sp_.config.mms_client_data_callback,
+                    arguments
+                );
+            } catch(e) {
+                console.log("mms_client_data_callback failed", e);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)